test/framework/universal_app.go

```diff
@@ -121,10 +151,15 @@ func NewUniversalApp(t testing.TestingT, clusterName string, mode AppMode, verbo
 
 	if mode == AppModeCP {
 		app.allocatePublicPortsFor("5678", "5680", "5681", "5682", "5685")
+	} else {
+		app.allocatePublicPortsFor("30001") // the envoy admin port
```

Comment:
why it was added?

Reply:
for debugging purposes, but I don't think it ever worked. Will remove it